src/State/Processor/RespondProcessor.php

```diff
@@ -129,7 +130,7 @@ public function process(mixed $data, Operation $operation, array $uriVariables =
                     $iri = $this->iriConverter->getIriFromResource($operation->getClass(), UrlGeneratorInterface::ABS_PATH, $operation);
                 }
 
-                if ($iri) {
+                if ($iri && !$operation instanceof Get) {
```

Comment:
```suggestion
                if ($iri && 'GET' !== $operation->getMethod()) {
```

Reply:
@soyuka Except that it also applies to `GetCollection` when it shouldn't? As a result, the CI crashes on `GetCollection` because it no longer contains the `Content-Location` header

Reply:
To me the test should be changed. There is no need for a `Content-Location` header if its value if the current URL and the method is POST.

I would consider this a fix, not a BC break.